legislators-current.yaml

```diff
@@ -40919,3 +40919,26 @@
     district: 1
     party: Republican
     phone: 202-225-2472
+- id:
+    bioguide: F000475
```

Comment:
That's not right. :)

Reply:
Sorry about that, good now?

Reply:
@putorti The `fecid` is the candidate ID, not the committee ID. Can you fix?

Reply:
@dwillis Sorry, my first new official; fewer mistakes next time!